src/libraries/System.Security.Cryptography.Pkcs/tests/SignedCms/SignedCmsTests.netcoreapp.cs

```diff
@@ -59,7 +59,7 @@ public static void SingUsingExplicitKeySetWithProperty()
             }
         }
 
-        [Fact]
+        [ConditionalFact(typeof(SignatureSupport), nameof(SignatureSupport.SupportsRsaSha1Signatures))]
```

Comment:
I don't see anything particularly "SHA1-y" about these tests. To keep these tests running on all platforms, I would:

1. Change `VerifyWithExplicitPrivateKey` at https://github.com/dotnet/runtime/blob/c4a15decfa2ab62256a7894cb962e7bcad278414/src/libraries/System.Security.Cryptography.Pkcs/tests/SignedCms/SignedCmsTests.netcoreapp.cs#L613 to something like:

    ```C#
    DigestAlgorithm = key is DSA ? new Oid(Oids.Sha1, Oids.Sha1) : new Oid(Oids.Sha256, Oids.Sha256)
     ```

    Basically, switch the test to SHA256 except for DSA, because DSA is old and should continue to be tested with SHA1.